spec/cucumber/formatter/html_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'cucumber/formatter/spec_helper'
+require 'cucumber/formatter/html'
+require 'cucumber/cli/options'
+
+module Cucumber
+  module Formatter
+    describe HTML do
+      extend SpecHelperDsl
+      include SpecHelper
+
+      before(:each) do
+        @out = StringIO.new
+        @formatter = HTML.new(actual_runtime.configuration.with_options(out_stream: @out, source: false))
+      end
+
+      describe 'with a scenario' do
+        define_feature <<-FEATURE
+          Feature: Banana party
+
+            Scenario: Monkey eats banana
+              Given there are bananas
+        FEATURE
+
+        before(:each) do
+          run_defined_feature
+        end
+
+        it 'outputs html' do
+          expect(@out.string).to include '<!DOCTYPE html>'
```

Comment:
If you want to be more rigorous, these could use the `start_with` and `end_with` matchers.

That is unless there is some other metadata (In which case ignore this comment).